packaging/windows/khiops_env.cmd

```diff
@@ -0,0 +1,156 @@
+echo If the --env flag is used, the environment list is printed in the standard output.
+echo.
+echo The following variables are used to set the path and classpath for the prerequisite of Khiops.
+echo.
+echo KHIOPS_PATH: path of Khiops executable, to add in path
+echo KHIOPS_MPI_COMMAND: MPI command to call the Khiops tool
+echo KHIOPS_JAVA_PATH: path of Java tool, to add in path
+echo KHIOPS_CLASSPATH: Khiops java libraries, to add in classpath
+echo.
+echo If they are not already defined, the following variables used by Khiops are set:
+echo.
+echo KHIOPS_LAST_RUN_DIR: directory where Khiops writes output command file and log
+echo   (when not defined with -e and -o)
+echo KHIOPS_PROC_NUMBER: processes number launched by Khiops (it's default value corresponds to the
+echo   number of physical cores of the computer plus one)
+echo.
+echo The following variables are not defined by default and can be used to change some default
+echo  properties of Khiops:
+echo.
+echo KHIOPS_TMP_DIR: Khiops temporary directory location (default: the system default).
+echo   This location can be modified from the tool as well.
+echo KHIOPS_MEMORY_LIMIT: Khiops memory limit in MB (default: the system memory limit).
+echo   The minimum value is 100 MB; this setting is ignored if it is above the system's memory limit.
+echo   It can only be reduced from the tool.
+echo KHIOPS_API_MODE: standard or api mode for the management of output result files created by Khiops
+echo   In standard mode, the result files are stored in the train database directory,
+echo   unless an absolute path is specified, and the file extension is forced if necessary.
+echo   In api mode, the result files are stored in the current working directory, using the specified results as is.
+echo   . default behavior if not set: standard mode
+echo   . set to 'true' to force standard mode
+echo   . set to 'false' to force api mode
+echo KHIOPS_RAW_GUI: graphical user interface for file name selection
+echo   . default behavior if not set: depending on the file drivers available for Khiops
+echo   . set to 'true' to allow file name selection with uri schemas
+echo   . set to 'false' to allow local file name selection only with a file selection dialog
+echo.
+echo In case of configuration problems, the variables KHIOPS_JAVA_ERROR and KHIOPS_MPI_ERROR contain error messages.
+
+if not %2.==. exit /b 1
+if %1.==-h. exit /b 0
+if %1.==--help. exit /b 0
+exit /b 1
+
+REM Set Khiops environment variables
+:DISPLAY_ENV
+setlocal
+set DISPLAY_ENV=true
+
+:SET_ENV
+REM Initialize exported variables
+set "KHIOPS_PATH="
+set "KHIOPS_COCLUSTERING_PATH="
+set "KHIOPS_MPI_COMMAND="
+set "KHIOPS_JAVA_PATH="
+set "KHIOPS_CLASSPATH="
+set "KHIOPS_JAVA_ERROR="
+set "KHIOPS_MPI_ERROR="
+
+REM Set Khiops home to parent directory
+for %%a in ("%~dp0..") do set "_KHIOPS_HOME=%%~fa"
+
+REM KHIOPS_PATH
+set "KHIOPS_PATH=%_KHIOPS_HOME%\bin\MODL.exe"
```

Comment:
Shouldn't we enclose the value we set "KHIOPS_PATH" to between quotes, so that they are not truncated at spaces? The same goes for all paths IMHO.